themes/index.js

```diff
@@ -355,6 +355,12 @@ const themes = {
     bg_color: "09131B",
     border_color: "0c1a25",
   },
+  Viðar: {
```

Comment:
@VidarDev Thanks for your contribution. Since your theme is distinct enough I'm happy to merge it. However, due to ease of use, I think it would be better to use a name without special characters. Maybe we can change this to `vidar` referring to the Norse god?

Reply:
yes do it